context.go

```diff
@@ -23,15 +23,17 @@ type Context struct {
 	RunAt     time.Time
 	Watcher   Watch
 
-	script            func(*Context) error
-	metadata          Metadata
-	latency           time.Duration
-	notUseLoop        bool
-	histogram         []*Histogram
-	indicator         []*Indicator
-	log               ILogger
-	context           context.Context //nolint:containedctx
-	contextCancelFunc context.CancelFunc
+	// Reminder: If add new fields change context.Copy function
```

Comment:
Só embelezando um pouco.
```suggestion
	// Reminder: If new fields are added, change context.Copy function accordingly.
```